tests/signup.spec.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { Provider } from 'react-redux';
+import store from '../src/store';
+// Component to be tested
+import Signup from '../src/components/compounds/Signup/Signup';
+import Modal from '../src/components/atoms/Modal/Modal';
+import formValid from '../src/util/helpers/formValid';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+
+describe('<Signup />', () => {
+  describe('render()', () => {
+    test('renders the component', () => {
+      const wrapper = mount(<Provider store={store}><Signup /></Provider>);
+
+      expect(wrapper.exists()).toBe(true);
+      expect(wrapper.find('form').exists()).toBe(true);
+      expect(wrapper.find('input').exists()).toBe(true);
+      expect(wrapper.find('button').exists()).toBe(true);
+      expect(wrapper.find('mediaButtonSection').exists()).toBe(true);
+    });
+  });
+
+  describe('render()', () => {
```

Comment:
describe text shouldn't be a function

Reply:
Also, test that clicking and all interactions work. This will require making use of an interceptor like `moxios`